packages/fxa-settings/src/components/ButtonBack/index.tsx

```diff
@@ -36,7 +57,7 @@ const ButtonBack = ({
       aria-label={localizedButtonAriaLabel}
       className="me-4 tablet:me-0 tablet:p-4 tablet:absolute tablet:-start-24 rounded focus-visible-default"
     >
-      <BackArrow className="w-6 h-auto text-grey-400 rtl:transform rtl:-scale-x-100" />
+      <BackArrow className={`w-6 h-auto text-grey-400 ${getTabletArrowColor()} rtl:transform rtl:-scale-x-100`} />
```

Comment:
Can we remove `text-grey-400` since we now calculate the best colour option?

Reply:
Good thinking but that's the color set for mobile, where we pull it "into" the card with a white background.